orc8r/cloud/docker/Makefile

```
@@ -0,0 +1,6 @@
+build:
+	./build.py --all --parallel
+run:
+	./build.py --all --parallel && ./run.py
+dev:
+	docker-compose down ; ./build.py --all --parallel && ./run.py ; docker-compose logs -f controller
```

Nit: always use the long form of flags in docs and checked-in code, as it provides easy self-documentation to future readers

```suggestion
	docker-compose down ; ./build.py --all --parallel && ./run.py ; docker-compose logs --follow controller
```